src/dos/program_loadfix.cpp

```diff
@@ -90,22 +90,26 @@ void LOADFIX::AddMessages() {
 	        "Load a program in the specific memory region and then run it.\n"
 	        "\n"
 	        "Usage:\n"
-	        "  [color=light-green]loadfix[reset] [color=light-cyan]GAME[reset] [color=white][PARAMETERS][reset]\n"
+	        "  [color=light-green]loadfix[reset] [color=white][-SIZE][reset] [color=light-cyan]GAME[reset] [color=white][PARAMETERS][reset]\n"
 	        "  [color=light-green]loadfix[reset] [/d] (or [/f])[reset]\n"
 	        "\n"
 	        "Parameters:\n"
-	        "  [color=light-cyan]GAME[reset]  game or program to load, optionally with parameters\n"
+	        "  [color=light-cyan]GAME[reset]\t\tgame or program to load, optionally with parameters\n"
+	        "  [color=white]-SIZE[reset]\t\tSIZE indicates the number of kilobytes to be allocated\n"
+	        "  /d (or /f)\tFrees the previously allocated memory."
```

Comment:
We don't do tabs (or at least we shouldn't 😅). Can you please change these to spaces.